Source/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -12,19 +12,32 @@ internal abstract class BuildableCallRule
         : IFakeObjectCallRule
     {
         private readonly List<Tuple<Func<IFakeObjectCall, bool>, Action<IOutputWriter>>> wherePredicates;
+        private Action<IInterceptedFakeObjectCall> applicator;
+        private bool canSetApplicator;
 
         protected BuildableCallRule()
         {
             this.Actions = new LinkedList<Action<IFakeObjectCall>>();
-            this.Applicator = call => call.SetReturnValue(call.Method.ReturnType.GetDefaultValue());
+            this.applicator = call => call.SetReturnValue(call.Method.ReturnType.GetDefaultValue());
+            this.canSetApplicator = true;
             this.wherePredicates = new List<Tuple<Func<IFakeObjectCall, bool>, Action<IOutputWriter>>>();
+            
         }
 
         /// <summary>
         /// Gets or sets an action that is called by the Apply method to apply this
         /// rule to a fake object call.
         /// </summary>
-        public Action<IInterceptedFakeObjectCall> Applicator { get; set; }
+        public Action<IInterceptedFakeObjectCall> Applicator
+        {
+            get { return this.applicator; }
+            set
+            {
+                if (!this.canSetApplicator) throw new InvalidOperationException("The behavior for this rule is already defined");

```

Comment:
I'd say probably not.
End users won't know what a "rule" is. Of course the stack trace should give a good hint at what went wrong, but I think it's preferable to be as specific as possible while using the end user API's language. And I'd be inclined to help guide the user to better behaviour. They're likely trying to set up behaviour that varies over time, and we could push them toward multiple calls to `A.CallTo` (or, unfortunately, `Fake.CallsTo`, so wording could become tricky).